package.json

```diff
@@ -29,14 +29,17 @@
     "@babel/plugin-transform-template-literals": "^7.8.3",
     "@babel/polyfill": "^7.0.0",
     "@babel/preset-env": "^7.0.0",
+    "@babel/preset-flow": "^7.8.3",
     "@babel/preset-react": "^7.0.0",
     "@babel/register": "^7.0.0",
+    "@khanacademy/eslint-config": "0.0.5",
```

Comment:
Wasn't this added in a previous PR?

Reply:
Yeah! This should be just re-ordering it